.github/workflows/release-swift.yml

```diff
@@ -0,0 +1,215 @@
+        id: set-release-name
+        run: |
+          if [[ ${{ inputs.pre-release }} == true ]]; then
+            echo "release_name=${{ env._PKG_VERSION }}-unstable-${{ needs.validate.outputs.short_sha }}" >> $GITHUB_OUTPUT
+          else
+            echo "release_name=${{ env._PKG_VERSION }}" >> $GITHUB_OUTPUT
+          fi
+
+  repo-sync:
+    name: Push changed files to SDK Swift repo
+    runs-on: ubuntu-22.04
+    needs: validate
+    env:
+      _BOT_EMAIL: 106330231+bitwarden-devops-bot@users.noreply.github.com
+      _BOT_NAME: bitwarden-devops-bot
+      _PKG_VERSION: ${{ needs.validate.outputs.version }}
+      _PRE_RELEASE: ${{ inputs.pre-release }}
+      _RELEASE_NAME: ${{ needs.validate.outputs.release_name }}
+    steps:
+      - name: Checkout SDK repo
+        uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4.0.0
+        with:
+          path: sdk
+
+      - name: Get ref from SDK repo
+        id: get-ref
+        run: |
+          if [[ $_PRE_RELEASE == true ]]; then
+            echo "ref=unstable" >> $GITHUB_OUTPUT
+          else
+            echo "ref=main" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Checkout SDK-Swift repo
+        uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4.0.0
+        with:
+          repository: bitwarden/sdk-swift
+          path: sdk-swift
+          ref: ${{ steps.get-ref.outputs.ref }}
+
+      - name: Login to Azure - Prod Subscription
+        uses: Azure/login@92a5484dfaf04ca78a94597f4f19fea633851fa2 # v1.4.7
+        with:
+          creds: ${{ secrets.AZURE_CI_SERVICE_PRINCIPAL }}
+
+      - name: Retrieve secrets
+        id: retrieve-secrets
+        uses: bitwarden/gh-actions/get-keyvault-secrets@main
+        with:
+          keyvault: ${{ env._KEY_VAULT }}
+          secrets: "github-pat-bitwarden-devops-bot-repo-scope"
+
+      - name: Setup Git
+        working-directory: sdk-swift
+        run: |
+          git config --local user.email "${{ env._BOT_EMAIL }}"
+          git config --local user.name "${{ env._BOT_NAME }}"
+
+      - name: Update files
+        run: |
+          # Update BitwardenFFI path
+          sed -i '' 's|.binaryTarget(name: "BitwardenFFI", path: "BitwardenFFI.xcframework")|.binaryTarget(\
+            name: "BitwardenFFI",\
+            url: "https://github.com/bitwarden/sdk-swift/releases/download/v${{ env._RELEASE_NAME }}/BitwardenFFI-${{ env._PKG_VERSION }}-${{ needs.validate.outputs.short_sha }}.xcframework.zip",|' sdk/languages/swift/Package.swift
+
+          # Run swiftformat
+          swiftformat sdk/languages/swift/Package.swift
+
+          # Copy files to local sdk-swift repo path
+          cp --verbose -rf sdk/languages/swift/README.md sdk-swift/README.md
+          cp --verbose -rf sdk/languages/swift/Package.swift sdk-swift/Package.swift
+          cp --verbose -rf sdk/languages/swift/Sources sdk-swift/Sources
+          cp --verbose -rf sdk/languages/swift/Tests sdk-swift/Tests
```

Comment:
Are we calculating the checksum somewhere?'

I usually have to edit these two lines
```
            url: "https://bwlivefronttest.blob.core.windows.net/sdk/de3b6e2-BitwardenFFI.xcframework.zip",
            checksum: "37e884d820907af85e788b5e0ab483bfc5aa0299e5d6b39865ad70659322a202"),
```

Reply:
I thought I did it... It's fixed now. :)